IBPSA/Resources/C-Sources/WeeklySchedule.h

```diff
@@ -0,0 +1,67 @@
+/*
+
+  This code implements a weekly schedule. Input text example:
+
+#some comment
+double  tab1(3,5) # another comment
+# blabla
+mon:0:0:10              -       3       1       -
+tue,thu:20:30:59        123     -       45      -
+wed,sat,sun             12      1       4       -
```

Comment:
Better use mo, tu, we, th, fr, sa, su, it is shorter and suffices.

Reply:
I prefer three letters since it's easier to interpret. It also adds more degrees of freedom in case we decide to add other keys like 'wda' and 'wee' to indicate week-day and weekend. Can we leave it like this?

Reply:
I am fine with the three letters (but in general need a bit more time for review as I run out of time yesterday).